llm.py

```diff
@@ -32,8 +32,8 @@ def __init__(self, name="gpt-3.5-turbo", use_cache=True, override_hparams={}):
         self.name = name
         if 'gpt' in name:
             self.model = OpenAIModel(name)
-        elif 'llama' in name:
-            self.model = LLAMAModel(name)
+        # elif 'llama' in name:
```

Comment:
You want to revert this?

Reply:
I prefer not to commit any untested changes. Let me conduct some testing on the llama before making any updates.

Reply:
I'm encountering errors with LLaMA, specifically: `self.model.hparams.update(override_hparams) resulting in an AttributeError: 'LLAMAModel' object has no attribute 'hparams'`. 

 Since students prefer to use LLaMA and it's an open-source model, I plan to address this issue by submitting a separate pull request. This will include thorough testing, along with comprehensive notes explaining the changes made.